lang/xml_gettext.py

```diff
@@ -56,7 +56,26 @@ def _start(self, *args, **kwargs):
             info = f'#. {s}\n'
             info += f'#. {l}\n'
             for k, v in parent[elem].attrib.items():
-                info += f"#.   {k}: {v}\n"
+                info += f"#. {k}: {v}\n"
+            po[text] = (
+                f'\n{info}'
+                f'#: {filename}:{elem.lineno}\n'
+                f'msgid "{text}"\n'
+                'msgstr ""\n'
+            )
+        if elem.tag in ("enum"):
```

Comment:
not sure if these needs to be picked up by `xml_gettext.py`
```
<ui unit=" ms" />
```

ms being short for milisecond, but can have a different translation in other languages

Reply:
Pushed 2 commits to support this.